### DISCUSSION TITLE
Performance Score Variations Between Mobile and Desktop

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10202] [Q&A]
I've noticed significant performance differences between my website's mobile and desktop versions. The performance score for mobile is 44, while for desktop it's 88. This behavior is consistent both on the live website and in my local environment.

Has anyone else encountered similar performance discrepancies? What could be potential causes, and how can I improve the mobile performance score?

Any suggestions or insights would be appreciated!

Screenshot of live website performance score:
![Screenshot_21-10-2024_191935_pagespeed web dev](https://github.com/user-attachments/assets/bbd4e9ac-cc05-419c-9c21-82f604016ab6)
![Screenshot_21-10-2024_192115_pagespeed web dev](https://github.com/user-attachments/assets/00d1de6e-7641-4bc1-804a-52fbc0b187d9)

Screenshot of website performance score in my local environment:

For mobile:
![Screenshot (563)](https://github.com/user-attachments/assets/b2b09d28-1214-4fa7-af60-deb0dc218d6c)

For desktop:
![Screenshot (564)](https://github.com/user-attachments/assets/8710484c-1b9f-41a0-a0e2-b09a9f59c7e0)

Thank you!

**Comment by amit-webkul** [comment]
Hello @sudeep2003 ,

As I understand your concern, you are experiencing performance issues with your mobile site.

You can improve performance by making some adjustments to your code. These issues are sometimes caused by factors like image ratios, missing aria-label attributes, etc.

I have attached a screenshot for reference. You can see in the image that Google Chrome's Lighthouse feature highlights the areas in your code that need improvement. Using this tool, you can manage and improve performance.

![image](https://github.com/user-attachments/assets/684858b5-f5d4-42a6-98de-1281b65a3054)

If you get any issues, feel free to ask.
